posthog/tasks/update_cache.py

```diff
@@ -52,6 +53,38 @@
 }
 
 
+def update_filters_hash_caches() -> None:
+    # get everything
+    # try to queue it behind redis lock
+    dashboard_tiles = (
+        DashboardTile.objects.exclude(dashboard__deleted=True)
+        .exclude(insight__deleted=True)
+        .exclude(insight__filters={})
+        .select_related("insight", "dashboard")
+        .order_by(F("dashboard__last_accessed_at").desc(nulls_first=True))
+    )
+    for tile in dashboard_tiles.iterator():
+        push_to_queue(tile.insight, tile.dashboard)
+
+    shared_insights = (
+        Insight.objects.filter(sharingconfiguration__enabled=True)
+        .exclude(deleted=True)
+        .exclude(filters={})
+        .order_by(F("last_modified_at").asc(nulls_first=True))
+    )
+    for insight in shared_insights.iterator():
+        push_to_queue(insight, None)
+
+
+def push_to_queue(insight: Insight, dashboard: Optional[Dashboard]) -> None:
+    cache_key, cache_type, payload = insight_update_task_params(insight, dashboard)
+    update_filters_hash(cache_key, dashboard, insight)
+    was_set = get_client().set(name=f"processing-{cache_key}", value=cache_key, nx=True, ex=180)
```

Comment:
180? Firstly this feels low, when we first queue up all dashboards and insights, I would imagine it would take a fair bit of time to crunch through all of this.

Secondly, should we pull it out to a const and give it a comment?

Reply:
I would argue rather leave it high and assume the best case (that the queue eventually gets around to it and unlocks it)

Reply:
Yeah, I couldn't make my mind up...